libs/ui/inputs/src/lib/autocomplete/autocomplete.component.ts

```diff
@@ -86,7 +99,7 @@ export class AutocompleteComponent implements OnInit, AfterViewInit, OnDestroy {
     this.subscription.unsubscribe()
   }
 
-  initInput(value: AutcompleteItem) {
+  updateInputValue(value: AutcompleteItem) {
```

Comment:
Not sure if it's fixed on master, same typo I noticed in other PR.

Reply:
> Not sure if it's fixed on master, same typo I noticed in other PR.

not fixed yet, thanks for the review